queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
This query will find animals with `'mon'` in between their names, what we should query for are animals whose names end with `mon`. The `%` symbol is a wildcard depending on where you use it.

- [ ] Please update query